api/src/main/resources/workbench.yaml

```diff
@@ -706,12 +706,16 @@ paths:
             $ref: "#/definitions/CriteriaListResponse"
 
   # Cohort Review  #######################################################################
-  /api/v1/cohortreview/participants/{cohortId}/{cdrVersionId}:
+  /api/v1/cohortreview/info/{cohortId}/{cdrVersionId}:
     get:
       tags:
          - cohortReview
-      description:  Returns a collection of participants that for the specified cohortId and cdrVersionId
-      operationId: "getParticipantCohortStatuses"
+      description: >
+        This endpoint will check for existence of a cohort review specified by cohortId and
```

Comment:
Is there any reason we couldn't just insert this row automatically on cohort creation, based on the CDR version of the workspace you're working in? (That might eliminate the need to have a get method which inserts a row if it's not already there...)